launch/test_setup.launch.py

```diff
@@ -1,18 +1,164 @@
-import launch
-import launch.actions
-import launch.substitutions
 import os
+
 from ament_index_python.packages import get_package_share_directory
-import launch_ros.actions
-from launch.actions import DeclareLaunchArgument
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, GroupAction, SetEnvironmentVariable
+from launch.conditions import IfCondition
+from launch.substitutions import (LaunchConfiguration,
+                                  PythonExpression)
+from launch_ros.actions import LoadComposableNodes, Node, PushROSNamespace, SetParameter
+from launch_ros.descriptions import ComposableNode, ParameterFile
+from nav2_common.launch import LaunchConfigAsBool, RewrittenYaml
+
+
+def generate_launch_description() -> LaunchDescription:
+    # Get the launch directory
+    bringup_dir = get_package_share_directory('neo_localization2')
+
+    namespace = LaunchConfiguration('namespace')
+    use_sim_time = LaunchConfigAsBool('use_sim_time')
+    autostart = LaunchConfigAsBool('autostart')
+    params_file = LaunchConfiguration('params_file')
+    use_composition = LaunchConfigAsBool('use_composition')
+    container_name = LaunchConfiguration('container_name')
+    container_name_full = (namespace, '/', container_name)
+    use_respawn = LaunchConfigAsBool('use_respawn')
+    log_level = LaunchConfiguration('log_level')
+
+    lifecycle_nodes = ['neo_localization']
```

Comment:
```suggestion
    lifecycle_nodes = ['neo_localization2_node']
```

Because we use this name for setting the parameters in all the robots config.